packages/mobile/src/send/SendAmount/index.tsx

```diff
@@ -152,31 +162,38 @@ function SendAmount(props: Props) {
   )
   const maxBalance = tokenInfo?.balance.minus(feeEstimate) ?? ''
   const maxInLocalCurrency = useTokenToLocalAmount(maxBalance, transferTokenAddress)
+  const maxAmountValue = showInputInLocalAmount
+    ? maxInLocalCurrency
+      ? maxInLocalCurrency?.toFixed()
+      : ''
+    : maxBalance.toFixed(TOKEN_MAX_DECIMALS)
```

Comment:
Calling the `toFixed` here is losing some precision. Should we do just `toFixed()` or keep it as a `BigNumber`?